01 Phone number format.rb

```diff
@@ -0,0 +1,19 @@
+# Takes in a phone number and checks whether it's in standard
+# format or not.
+def is_valid_phone_number(num)
+  # Looks for  brackets ( and ), space and a hyphen at the 
+  # required positions.
+  if num[0] == "(" && num[1] != "0" && num[4] == ")" && 
+     num[5] == " " && num[9] == "-" && num.count("0-9") == 10 &&
+     num.size < 15
+     return true

```

Comment:
you don't need to return. the last statement in the method always gets returned in ruby

```suggestion
    true

```

But now after the above change you will see that the only thing in the if condition is true and false. Which is the same value that the if condition evaluates to. So you can basically just do: 

```
def is_valid_phone_number(num)
    num[0] == "(" && num[1] != "0" && num[4] == ")" &&  num[5] == " " && num[9] == "-" && num.count("0-9") == 10 && num.size < 15
end
```